README.md

```diff
@@ -64,9 +67,19 @@ Daily Builds
 [win-x64-badge-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/sharedfx_win-x64_Release_version_badge.svg
 [win-x64-version-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/latest.sharedfx.win.x64.version
 [win-x64-installer-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/dotnet-win-x64.latest.exe
+[win-x64-installer-checksum-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/release/2.0.0/dotnet-win-x64.latest.exe
 [win-x64-zip-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/dotnet-win-x64.latest.zip
+[win-x64-zip-checksum-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/dotnet-win-x64.latest.zip.sha512
 [win-x64-symbols-zip-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/dotnet-sharedframework-symbols-win-x64.latest.zip
 
+[win-x64-badge-2.X.X]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/release/2.0.0/sharedfx_win-x64_Release_version_badge.svg
+[win-x64-version-2.X.X]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/release/2.0.0/latest.sharedfx.win.x64.version
```

Comment:
This link doesn't exist.

Can you manually ensure each link you are adding works?